build.fsx

```diff
@@ -301,6 +314,7 @@ Target "All" DoNothing
 "Clean"
   =?> ("BuildVersion", isAppVeyorBuild)
   ==> "AssemblyInfo"
+  ==> "VsixManifest"
```

Comment:
We should only run this on AppVeyorCI. Locally, we don't want FAKE build to override our vsix version.

I think a better way is to extract Version value (e.g. 2.1.3) and append build numbers into it?